public/components/security/roles-mapping/components/roles-mapping-table.tsx

```diff
@@ -86,20 +91,23 @@ export const RolesMappingTable = ({ rolesEquivalences, rules, loading, editRule,
     {
       field: 'id',
       name: 'Status',
-      render (item, obj){
-        if(WzAPIUtils.isReservedID(item)){
-          if( (obj.id === 1 || obj.id === 2)){
-            return(
+      render(item, obj) {
+        if (WzAPIUtils.isReservedID(item)) {
+          if (obj.id === 1 || obj.id === 2) {
```

Comment:
(obj.id === 1 || obj.id === 2) podría ponerse algunas constantes que te indiquen qué significa el 1 y el 2.

Reply:
These changes were caused by the code formatting, not intentionally.

Anyways, you are right that using constants improves readability.